convert2rhel/main.py

```diff
@@ -41,6 +41,9 @@ class ConversionPhase(object):
 def main():
     """Perform all steps for the entire conversion process."""
 
+    # handle command line arguments
+    toolopts.CLI()
+
```

Comment:
Maybe pass this line after `utils.require_root()` ?  Since we need to check root permission before running anything...

Reply:
My logic is that cli should be available always. But I added https://github.com/oamg/convert2rhel/pull/176/commits/3d76e0f4d1ba2d958216f364f1754f3d3a1a2451 . 
What do you think guys @oamg/convert2rhel ?

Reply:
Agreed with @ZhukovGreen.